app/assets/stylesheets/application.css

```diff
@@ -1,4 +1,4 @@
-* {
+/* * {
```

Comment:
- [ ] [OPTIONAL] Please do not leave commented pieces of code in your codebase as it makes your work look messy and unprofessional. Use git history if you need to see a previous version of your project.